agents/hermes/mcp-config-transaction.py

```
@@ -680,6 +680,23 @@ def apply_transaction_and_reload(
+        try:
+            integrity_path = (
+                STRICT_HASH_PATH if privileged else os.path.join(HERMES_DIR, ".config-hash")
+            )
+            committed_text, _ = guard._read_text(CONFIG_PATH)
+            if (
+                committed_text == expected_text
+                and guard.inspect_mcp_integrity(HERMES_DIR, integrity_path) == "current"
+            ):
+                return {"ok": True, "changed": True, "reloaded": True}
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)